examples/Cart/CartAddLineItemAction.json

```diff
@@ -1,5 +1,34 @@
 {
   "action": "addLineItem",
   "productId": "{{productId}}",
-  "variantId": 1
+  "variantId": 2,
+  "quantity": 1,
+  "supplyChannel": {
+    "typeId": "channel",
+    "id": "{{channelId}}"
+  },
+  "distributionChannel": {
+    "typeId": "channel",
+    "id": "{{channelId}}"
+  },
+  "externalTaxRate": {
+    "name": "StandardExternalTaxRate",
+    "amount": 0.19,
+    "country": "DE",
+    "state": "Bavaria"
+  },
+  "externalPrice": {
+    "currencyCode": "EUR",
+    "centAmount": 4000
+  },
+  "externalTotalPrice": {
+    "currencyCode": "EUR",
+    "centAmount": 4000
+  },
+  "shippingDetails": [
```

Comment:
Same here with multiple shipping addresses

Reply:
This one is for demo purposes. I would like to leave it as is.